src/commands/dev/mod.rs

```diff
@@ -84,6 +85,18 @@ pub fn dev(
     })
 }
 
+fn print_alpha_warning() {
+    println!(
+        "{} wrangler dev is currently unstable and there are likely to be breaking changes.",
```

Comment:
```suggestion
        "{} wrangler dev is currently in development.",
```

Reply:
I think this is ok the way it is - don't want to misrepresent what the dealio is